src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,62 @@
+.clock {
+  @extend %radius;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 660px;
```

Comment:
Use 'width: 80vmin;' and 'height: 80vmin;' instead of fixed pixel values to make the clock size responsive according to the task description.

Reply:
done